react-form-controls/my-app/src/RegistrationFormControlled.js

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+
+export default function UncontrolledForm() {
+  const [username, setUserName] = useState('');
+  const [password, setPassword] = useState('');
+  function handleSubmit(e) {
+    e.preventDefault();
+    console.log({ username, password });
+  }
+  return (
+    <form onSubmit={handleSubmit}>
+      <label for="username">
```

Comment:
In React, you have to use `htmlFor` instead of `for`. However, neither is needed if you wrap the `<label>` around the input as you have done here.